docs/Drawer.md

```diff
@@ -66,7 +66,7 @@ const xsDown = useMediaQuery(theme.breakpoints.down('xs'));
 | ------------------------------- | ---------------------------------------------------------------------------- | ------------------------------------------------------------ | -------- | ------- |
 | activeItem                      | itemID for the 'active' item                                                 | `string`                                                     | no       |         |
 | classes                         | Style overrides                                                              | `DrawerClasses`                                              | no       |         |
-| condensed                       | Use the smaller width drawer without text labels (rail variant only)         | `boolean`                                                    | no       | `false` |
+| condensed                       | Sets a smaller width when the drawer is using the rail variant               | `boolean`                                                    | no       | `false` |
```

Comment:
I prefer some of the extra detail from the previous description (without labels is kind of an important part of the condensed variation)